src/server/index.ts

```diff
@@ -60,8 +69,36 @@ const MORGAN_LOG_FORMAT =
   ':client-ip - [:date[clf]] ":method :url HTTP/:http-version" :status ' +
   '":redirectUrl" ":userId" :res[content-length] ":referrer" ":user-agent" :response-time ms'
 
+const connectSrc = ["'self'", 'www.google-analytics.com']
+if (cspReportUri) {
+  connectSrc.push(parseDomain(cspReportUri))
+}
+if (sentryDns) {
+  connectSrc.push(sentryDns)
+}
+
 const app = express()
 app.use(helmet())
+app.use(
+  helmet.contentSecurityPolicy({
+    directives: {
```

Comment:
could we also include `frameSrc`, `frameAncestors` to protect against frame injection XSS?

Reply:
Wouldn't `frameSrc` fallback on `defaultSrc` if not specified? I'll look into frameAncestors now